homeassistant/components/bluetooth/websocket_api.py

```diff
@@ -148,3 +149,36 @@ async def ws_subscribe_advertisements(
     _AdvertisementSubscription(
         hass, connection, msg["id"], BluetoothCallbackMatcher(connectable=False)
     ).async_start()
+
+
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "bluetooth/subscribe_connection_allocations",
+        vol.Optional("source"): str,
```

Comment:
After doing https://github.com/home-assistant/frontend/pull/23877 instead of source lets go with `config_entry_id` since thats what we are going to have. If we want to support source we can always make it vol.Exclusive later

Reply:
changed my mind on this https://github.com/home-assistant/core/pull/136215#issuecomment-2613741560